.gitignore

```diff
@@ -56,3 +56,5 @@ stan.kdev4
 
 # local make include
 /make/local
+
+*.gch
```

Comment:
[optional]

Looks like this file is missing a newline.  I don't know if that's a big deal or not.

Reply:
fixed